src/gui.js

```diff
@@ -3609,7 +3640,7 @@ IDE_Morph.prototype.settingsMenu = function () {
             }
             */
             Process.prototype.enableJS = !Process.prototype.enableJS;
-            this.currentSprite.blocksCache.operators = null;
+            this.currentSprite.primitivesCache.operators = null;
```

Comment:
Technically speaking these were not necessary changes, but seemed more clear. (Since I was originally trying to stuff custom blocks cache in here, and that actually got a little messy...)